StreamingAnalytics/ResourceMonitoring/ResourceMonitoring/config/connectivity/MQTT/MQTT.properties

```diff
@@ -0,0 +1,36 @@
+# $Copyright (c) 2016,2018 Software AG, Darmstadt, Germany and/or Software AG USA Inc., Reston, VA, USA, and/or its subsidiaries and/or its affiliates and/or their licensors.$
+# Use, reproduction, transfer, publication or disclosure is prohibited except as specifically provided for in your License Agreement with Software AG
+#
+# This file contains the main properties that need configuring to use this connectivity plug-in. 
+#
+# The properties defined here are used to replace ${...} substitution variables in the YAML file(s). 
+# For more advanced transport and codec configuration, see the associated YAML file. 
+# Backslash characters in property values must be escaped (e.g. \\). 
+#
+
+###########################################################################################
+# MQTT connectivity plug-in configuration for sending/receiving events to/from an MQTT
+# broker. Codecs will need to be configured in the corresponding YAML file.
+###########################################################################################
+
+# Use this setting to specify the URL of the broker you want to use for MQTT connectivity.
+MQTT_brokerURL=localhost:1883
```

Comment:
This can just be set to 'localhost' as it is for the other examples.

Reply:
Done